### PR TITLE
Improve/update

### PR DESCRIPTION
**Pull request by sepandasadi**

### CODE REVIEW

**Comment by sepandasadi** [comment]
@isuttell @corescan Please let me know if you are moving forward with this component so I can fix the tests to pass, or delete the repo. Thanks

File: README.md [hunk 1]
```diff
@@ -3,8 +3,8 @@
 
 [![npm](https://img.shields.io/npm/v/ship-components-highlight-click.svg?maxAge=2592000)](https://www.npmjs.com/package/ship-components-highlight-click)
 [![Build Status](http://img.shields.io/travis/ship-components/ship-components-highlight-click/master.svg?style=flat)](https://travis-ci.org/ship-components/ship-components-highlight-click)
-[![Code Quality](https://img.shields.io/codeclimate/github/ship-components/ship-components-highlight-click.svg?style=flat)](https://codeclimate.com/github/ship-components/ship-components-highlight-click)
```
Comment (isuttell) [inline README.md]: Why'd you remove the code quality badge?

File: tasks/config/eslint.js [hunk 2]
```diff
@@ -10,6 +10,11 @@
  */
 module.exports = function(grunt) {
   grunt.config.set('eslint', {
-    target: ['src/**/*.+(jsx?|es6)']
+    target: ['src/*.jsx'],
```
Comment (isuttell) [inline tasks/config/eslint.js]: Let's leave it at the previous glob so it's grabs js files too

**Review by isuttell — DISMISSED** [review]

File: tasks/config/eslint.js (hunk 2 above)
Comment (sepandasadi) [reply tasks/config/eslint.js]: Fixed

File: README.md (hunk 1 above)
Comment (sepandasadi) [reply README.md]: Will add back soon.

File: tasks/config/eslint.js [hunk 3]
```diff
@@ -10,6 +10,11 @@
...
 module.exports = function(grunt) {
   grunt.config.set('eslint', {
-    target: ['src/**/*.+(jsx?|es6)']
+    target: ['src/*.js', 'src/*.jsx' , 'src/*.es6'],
+    options: {
+      maxWarnings: 0
+    }
   });
+
+  grunt.loadNpmTasks('grunt-eslint');
```
Comment (isuttell) [inline tasks/config/eslint.js:19]: If you're using `jit-grunt`, I don't think we need to manually load the module like this. Can you verify that?

**Review by isuttell — CHANGES_REQUESTED** [review]

File: tasks/config/eslint.js (hunk 3 above)
Comment (sepandasadi) [reply tasks/config/eslint.js:19]: It throws an error (Unknown lint Task) when testing inside the Travis env. This will fixes it.

**Review by isuttell — APPROVED** [review]

**Pull request merged**